tests/foreman/ui/test_host.py

```diff
@@ -251,16 +251,22 @@ def module_activation_key(module_org_with_manifest, module_target_sat):
     return activation_key
 
 
-@pytest.fixture(scope='function')
-def remove_vm_on_delete(target_sat, setting_update):
-    setting_update.value = 'true'
-    setting_update.update({'value'})
-    assert (
-        target_sat.api.Setting().search(query={'search': 'name=destroy_vm_on_host_delete'})[0].value
-    )
-    yield
+@pytest.fixture
+def tracer_install_host(rex_contenthost, target_sat):
```

Comment:
I see the def is replacing another def but doesnt seems we are chaning the usage of this replaced function.

Expected ?

Reply:
Oh, I missed this while resolving conflicts, removed it, Thanks!